rpcs3/rpcs3qt/debugger_frame.cpp

```diff
@@ -753,8 +756,8 @@ void debugger_frame::keyPressEvent(QKeyEvent* event)
 			default: break;
 			}
 
-			if (const usz pos = std::basic_string_view<u32>(res.data(), 2).find_last_not_of(umax); pos != umax)
-				m_debugger_list->ShowAddress(res[pos] - std::max(row, 0) * 4, true);
+			if (auto it = std::find_if_not(res.rbegin(), res.rend(), std::bind(std::equal_to<u32>(), _1, umax)); it != res.rend())
```

Comment:
please use a lambda instead of std::bind

Reply:
With the godsend FN macro

Reply:
I hate that macro XD
terrible design decision